frontend/js/src/cb-review/CBReviewModal.tsx

```diff
@@ -694,7 +695,7 @@ export default NiceModal.create((props: CBReviewModalProps) => {
               <img
                 src="/static/img/critiquebrainz-logo.svg"
                 height="30"
-                className="cb-img-responsive"
+                className="cb-img-fluid"
```

Comment:
This class does not exist, but then again neither does `cb-img-responsive`.
 Maybe leftover from some old code?
 In any case, can probably be removed.